proto/src/determined/api/v1/model.proto

```diff
@@ -49,10 +53,24 @@ message GetModelsRequest {
   string name = 5;
   // Limit the models to those matching the description.
   string description = 6;
+
+  // Limit the models to those with the following labels.
+  repeated string labels = 7;
+
+  // Limit to unarchived models only.
+  google.protobuf.BoolValue archived = 8;
```

Comment:
+1. The underlying BoolValue class makes it nillable, whereas a naked bool isn't nillable in Go. But in the protobuf definition bool *is* nillable.

Reply:
This actually needs to store 3 values - if its `true` only show `archived` models, if its `false`, only show `unarchived` models, and if its null, show all regardless of `archive` status. If we made it a primitive `bool` it would default to `false` instead of `null`, which is not what we want. This is how the `experiment.proto` file is done as well.